internal-packages/react-graphql-test-app/src/components/DisplayPerson.tsx

```diff
@@ -1,7 +1,11 @@
-import { useQuery } from '@data-eden/react';
+import { useQuery, graphql } from '@data-eden/react';
+import {
+  type PersonQuery,
+  type PersonFieldsFragment,
+  type PersonQueryVariables,
+} from './__generated/DisplayPerson.graphql.js';
```

Comment:
This seems like a step backwards. Why go back to a top level `__generated` dir? It seems better to emit a sibling file to this one named `DisplayPerson.graphql.d.ts`...

Reply:
That said, where the actual generated files go is something we can definitely tweak going forward, but I'm gonna leave it as is for now.